### ISSUE TITLE
Rule idea: variable-name-space-after

### ISSUE THREAD

**Issue by bjankord** [kristerkari/stylelint-scss#52]
This rule idea comes from scss-lint's [Space After Variable Name rule](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#spaceaftervariablename)

From the scss-lint docs:

_Variables should be formatted with no space between the name and the colon._

**Bad: space before colon**
```scss
$my-var : 0;
```

**Good**
```scss
$my-var: 0;
```

**Comment by dryoma** [comment]
Thanks, definitely a must.

In the meantime, stylelint's [`declaration-colon-space-before`](http://stylelint.io/user-guide/rules/declaration-colon-space-before/) and other `declaration-colon-` rules could help a bit, with variables declared inside rulesets.

**Comment by jeddy3** [comment]
> In the meantime, stylelint's `declaration-colon-space-before` and other `declaration-colon-*` rules could help a bit, with variables declared inside rulesets.

It's interesting that `declaration-colon-space-before` only ignores unnested dollar variables at the moment. You could argue that it is should ignore them all, and that it is an oversight that it doesn't already. Either way, I doubt we'll change the behaviour until you guys have:

- `dollar-variable-colon-space-after`
- `dollar-variable-colon-space-before`

Which would check the spaces around dollar variable colons, whether nested or not. As, as you said, it's useful at the moment :)

**Comment by dryoma** [comment]
Published with 1.3.0.
